src/acme/pd/Ticket.java

```diff
@@ -96,10 +96,12 @@ private void calcEstimatedTimes() {
       LocalDateTime resultOfCouriersAndMiles = deliveryTime.minus((long)(60*timeToTravel), ChronoUnit.MINUTES);
       resultOfCouriersAndMiles.plusMinutes(5);
       
-      if(resultOfCouriersAndMiles.isBefore(LocalDateTime.now()))
-      {
-        this.estimatedDepartureTime = resultOfCouriersAndMiles;
-      }
+     
+      try {
+        if(resultOfCouriersAndMiles.isAfter(LocalDateTime.now()))
```

Comment:
I changed this in one of my branches as well, I ran into a null value for estimated departure time